src/System.Net.Requests/tests/FtpWebRequestTest.cs

```diff
@@ -70,7 +70,7 @@ public void GetResponse_ServerNameNotInDns_ThrowsWebException()
         public void GetResponse_ConnectFailure_ThrowsWebException()
         {
             // This is an invalid IP address, so we should fail to connect.
-            string serverUrl = "ftp://192.0.2.1/";
+            string serverUrl = "ftp://127.0.0.1/";
```

Comment:
>You can try `ftp://serverthatshouldnotexist/" or similar.

If you want to have a DNS name that doesn't exist, you should use code like `new Guid()` to generate a name that will be unique (and thus not exist).